data/Identifying_production_data_types.md

```diff
@@ -1,3 +1,216 @@
 # Data types found in production service repos
 
+---
 
+#Expression
+##Description this stores expression data of a variety of different types (microarray, RNA-Seq, Proteomics, q-PCR)
+
+##Methods
+These will need to be changed and rewritten.  Currently it is all written in Perl and acesses the data via direct SQL queries against the Central Store.Also many of the methods were geared for searching before search existed.  The search related methods will not be carried over and should be abandoned.  The currented outdated methods exist here: https://github.com/kbase/expression/blob/master/lib/Bio/KBase/KBaseExpression/KBaseExpressionImpl.pm
+I envision the following new methods being done and operating on the workspace objects:
+* Visualize Sample
+* Visualize Series
+* Add annotation
+* Get Data By SampleIDs and FeatureIDs
+* Visualize ReplicateGroup?
+* Not sure what the RNA-Seq people will need.  Currently there are no methods for them in this spec.
+* Visualize the expression data for a genome (by samples)
+* Visulaize the expression data for a list of features (by samples)
+* Compare Samples (define denominator as specific sample or average of a group of samples)
+* Average Samples together to make a representative Sample
+* Compare Samples vs default controls?
+* Determine ON and OFF calls
+* Get Top Changers
+* Make new sample with original values
+
+##Data
+The data is currently being stored in both the Central Store and the Workspace.  They are not dependent on one another.
+However for uploading from GEO the current pipeline requires getting information from the Genomes and Features in the CS.
+
+##Owner/Author
+Jason Baumohl
+
+##Workspace
+KBasePublicExpression
+
+##Repo
+https://github.com/kbase/expression
+
+##Data Sources
```

Comment:
This is meant to be a list of data source dependencies required to operate the service or complete a method.  So the KBase Workspace service would be one (user data, etc), and the KBase Central Store.  What else?  Where is it fetched from?  Who should we talk to for more info or do you know?